src/common/target.c

```diff
@@ -168,7 +168,6 @@ static const TargetEntry TargetMap[] = {
     {   "sim6502",      TGT_SIM6502     },
     {   "sim65c02",     TGT_SIM65C02    },
     {   "supervision",  TGT_SUPERVISION },
-    {   "vc20",         TGT_VIC20       },
```

Comment:
Just because it is undocumented does not mean noone is using it.

Yes, do not introduce explicit aliases. But, if they are already there, do not remove them without the need. Do not break the work of others just for the sake of breaking it.